docs/about.md

```diff
@@ -34,3 +34,21 @@ Talawa’s desired main features are:
 1. Donation acceptance
 1. Service provider level multi-organization capability
 1. Plugin support for administration services such as billing
+
+### Technologies Used
+
+Technology plays an increasingly significant role in improving access to education for people living in impoverished areas and developing countries. Some of the technologies that are being used by Talawa are:
```

Comment:
The first sentence is exaggerated. We should be more concise and straightforward since Talawa features have been discussed already in the docs.